pages/settings/wallets/SetupWallet.tsx

```diff
@@ -28,6 +28,9 @@ import { REQUIRED_CAPABILITIES } from "~/lib/constants";
 import { errorToast } from "~/lib/errorToast";
 
 export function SetupWallet() {
+  const { nwcUrl } = useLocalSearchParams<{
```

Comment:
I think the naming is confusing now with `nwcUrl` and `nostrWalletConnectUrl` in the same component. Could this new one be clearly identified e.g. `nwcUrlFromSchemeLink` or something?

Reply:
Done!